alembic/versions/ea5470295689_sequencing_stats.py

```diff
@@ -0,0 +1,42 @@
+"""Add sequencing-stats table
+
+Revision ID: ea5470295689
+Revises: 9008aa5065b4
+Create Date: 2023-05-04 09:29:09.945895
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy import Column, types, orm, ForeignKey
+
+# revision identifiers, used by Alembic.
+revision = "ea5470295689"
+down_revision = "9008aa5065b4"
+branch_labels = None
+depends_on = None
+
+Base = declarative_base()
+
+
+def upgrade():
+    op.create_table(
+        "sequencing_stats",
+        sa.Column("id", sa.Integer(), nullable=False),
+        sa.Column("sample_id", sa.Integer(), nullable=False),
+        sa.Column("flow_cell_id", sa.Integer(), nullable=False),
+        sa.Column("flow_cell_name", sa.String(), nullable=False),
+        sa.Column("sample_internal_id", sa.String(), nullable=False),
+        sa.Column("lane", sa.Integer(), nullable=False),
+        sa.Column("yield_mb", sa.Integer(), nullable=False),
```

Comment:
What yield is this? Sequenced bases, generated bytes? Should we have the unit in the name? What is the practice here?

Reply:
So. I checked the illumina website, since this was a bit unclear. 

Yield is the amount of data in bases. In CG stats pf_yield, which I think is "passing filter" yield is converted to yield in megabases.